src/stacks-hierarchy/Root_PurchaseConfirmationScreen/navigation-types.ts

```diff
@@ -1,5 +1,5 @@
 import {TicketRecipientType} from '@atb/ticketing';
-import {PurchaseSelectionType} from '@atb/stacks-hierarchy/types.ts';
+import {PurchaseSelectionType} from "@atb/purchase-selection";
```

Comment:
before it was a single quote ', now is double ", maybe to follow how it is in other places.